src/app.tsx

```diff
@@ -66,6 +66,9 @@ export const App = () => {
 					createRoutesFromChildren,
 					matchRoutes,
 				}),
+				Sentry.feedbackIntegration({
+					colorScheme: "system",
+				}),
```

Comment:
Do we need it if we create a custom feedback form?

Reply:
removed